<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Manjus-iMac.local" failures="1" tests="4" name="testing.reporter" time="5.135" errors="0" timestamp="2023-07-02T20:28:16 IST" skipped="0">
  <testcase classname="testing.reporter" name="reportertest1" time="5.130">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testing.reporter.reportertest1(reporter.java:15)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
    <system-out>
      <![CDATA[method name-reportertest1
 <a href ="//Users//manju//eclipse-workspace//testing//screenshot//Sat-Jul-01-19-03-43-IST-2023.png⁩"> Test Result <a/>]]>
    </system-out>
  </testcase> <!-- reportertest1 -->
  <testcase classname="testing.reporter" name="reportertest2" time="0.002">
    <system-out>
      <![CDATA[method name-reportertest2
this is test2
status1]]>
    </system-out>
  </testcase> <!-- reportertest2 -->
  <testcase classname="testing.reporter" name="reportertest4" time="0.001">
    <system-out>
      <![CDATA[method name-reportertest4
status1]]>
    </system-out>
  </testcase> <!-- reportertest4 -->
  <testcase classname="testing.reporter" name="reportertest3" time="0.002">
    <system-out>
      <![CDATA[method name-reportertest3
status1]]>
    </system-out>
  </testcase> <!-- reportertest3 -->
  <system-out>
    <![CDATA[method name-reportertest1
 <a href ="//Users//manju//eclipse-workspace//testing//screenshot//Sat-Jul-01-19-03-43-IST-2023.png⁩"> Test Result <a/>
method name-reportertest2
this is test2
status1
method name-reportertest3
status1
method name-reportertest4
status1]]>
  </system-out>
</testsuite> <!-- testing.reporter -->
